src/js/containers/App/App.js

```diff
@@ -25,41 +25,54 @@ class AppContainer extends Component {
-                    </LayoutRow>
-                    <LayoutRow>
-                        <LayoutColumn width={1/4}>
-                            { isLoggedIn() ? <ActiveZoneSelector/> : <noscript/> }
-                        </LayoutColumn>
-                        <LayoutColumn width={2/4}>&nbsp;</LayoutColumn>
-                        <LayoutColumn width={1/4}>
-                            { (isLoggedIn() && this.props.state.zoneSettings.entities[this.props.state.activeZone.id]) ? <UnderAttackButton/> : <noscript/> }
-                        </LayoutColumn>
-                    </LayoutRow>
-                    <LayoutRow>
-                        <LayoutColumn width={1/3}>&nbsp;</LayoutColumn>
-                        <LayoutColumn width={1/3}>
-                            <AppNavigation />
-                        </LayoutColumn>
-                        <LayoutColumn width={1/3}>&nbsp;</LayoutColumn>
-                    </LayoutRow>
-                    <LayoutRow>
-                        <LayoutColumn width={1/5}>&nbsp;</LayoutColumn>
-                        <LayoutColumn width={3/5}>
-                            {this.props.children}
-                        </LayoutColumn>
-                        <LayoutColumn width={1/5}>&nbsp;</LayoutColumn>
-                    </LayoutRow>
-                    <LayoutRow>
-                        <LayoutColumn width={1/1}>
-                            <p style={{ 'textAlign': 'center' }}><FormattedMessage id="container.App.version" values={{ 'version': this.props.state.config.config.version }}/></p>
-                        </LayoutColumn>
-                    </LayoutRow>
+                    <div style={{ border: "0.5px solid #ddd", marginTop: "8px", borderTop: "none" }}>
+                        <LayoutRow>
+                            <header id="header" className="header app-header">
+                                <div className="gradient-bar-header">
+                                    <div className="header-main" style={{ borderBottom: "1px solid #ddd", backgroundColor: "#f9f9f9" }}>
+                                        <LayoutColumn width={3/8}>
+                                            <LayoutColumn width={3/8}>
+                                                <img style={{ margin: "4.95px auto 0", width: "170px", height: "30px" }} src={ getAbsoluteUrl(config, 'assets/logo.svg') } />
+                                            </LayoutColumn>
+                                            <LayoutColumn width={5/8}>
+                                                { isLoggedIn() ? <ActiveZoneSelector/> : <noscript/> }                                            
+                                            </LayoutColumn>
+                                        </LayoutColumn>     
+                                        <LayoutColumn width={5/8}>
+                                            <div style={{ float: "right" }}>
```

Comment:
Technically `<Text align="right">` adds `text-align: right` instead of `float: right` but it doesn't matter in this case.